### PR TITLE
fix for date issue

### PR DESCRIPTION
**Pull request by nickbristow**
## Background
Provide a brief overview of the work completed here.

## GitHub Issue
Replace this text with the GitHub issue link.
**NOTE: Please don't mark the issue as closed by this pull request!  Issues are to be closed manually within Zenhub upon merge & prod deploy.**

## Associated PRs
If applicable, replace this text with links to additional pull requests associated with this work.

## Validation Plan
Please outline the steps you've already taken to validate this body of work, along with any additional steps that need to be taken in order to complete testing.

## Tech Specs
- [ ] If any TypeORM entities were created/updated/removed, the necessary migration(s) were [correctly auto-generated](https://github.com/typeorm/typeorm/blob/master/docs/using-cli.md#generate-a-migration-from-existing-table-schema) and **not manually written**.

## Automated Testing
- [ ] All unit tests are passing.
- [ ] Unit tests have been added to cover this changeset.

## Additional Context
If there's any additional information that those reviewing or validating this work should be aware of, please specify it here.

### CODE REVIEW

**Review by lukenewman — APPROVED** [review]

**Review by gordonfarrell — APPROVED** [review]
Nice!

**Comment by lukenewman** [comment]
this pr fixes https://github.com/ctoec/data-collection/issues/1605 and https://github.com/ctoec/data-collection/issues/1520

**Comment by bamader** [comment]
Imma validate this bad boi since it's blocking other stuff and helps clear the board.

## Validation Plan
Log into the app provider side.

- [ ] Uploaded spreadsheet of child records correctly saves birthday and enrollment/funding dates (they need to match the sheet and the preview)
- [ ] Changes to birthday correctly persist after both refreshing and navigating back to roster
- [ ] Changes to enrollment dates correctly persist after both refreshing and navigating back to roster
- [ ] Changes to funding dates correctly persist after both refreshing and navigating back to roster

File: src/entity/transformers/momentTransformer.ts [hunk 1]
```diff
@@ -6,6 +6,8 @@ export const momentTransformer = {
   to: (entityDate?: Moment) => {
     if (entityDate instanceof FindOperator) return entityDate;
     if (typeof entityDate === 'string') return entityDate;
-    return entityDate ? entityDate.format('YYYY-MM-DD') : undefined;
+    return entityDate
+      ? entityDate.utc().local(true).format('YYYY-MM-DDTHH:mm:ss')
```
Comment (jessethach) [inline src/entity/transformers/momentTransformer.ts:10]: Weird that it needs the extra timestamp formatting. Looks like `.local` did the trick!

**Review by jessethach — APPROVED** [review]

**Pull request merged**